src/main/java/pl/betoncraft/betonquest/exceptions/UnsupportedVersionException.java

```diff
@@ -33,11 +29,9 @@
      * @param requiredVersion The expected version
      */
     public UnsupportedVersionException(final Plugin plugin, final String requiredVersion) {
-        super(String.format("%s version %s is not supported. Please install version %s!",
-                plugin.getName(),
+        super(plugin, String.format("Version %s is not supported. Please install version %s!",
```

Comment:
Why remove the plugin name? Reading the message now doesn't tell which plugin's version isn't supported?